specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json

```diff
@@ -0,0 +1,842 @@
+        }
+      ],
+      "description": "The storage task assignment."
+    },
+    "StorageTaskAssignmentProperties": {
+      "type": "object",
+      "properties": {
+        "taskId": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "Id of the corresponding storage task"
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether the storage task assignment is enabled or not"
+        },
+        "description": {
+          "type": "string",
+          "description": "Text that describes the purpose of the storage task assignment"
+        },
+        "executionContext": {
+          "$ref": "#/definitions/StorageTaskAssignmentExecutionContext",
+          "description": "The storage task assignment execution context"
+        },
+        "report": {
+          "$ref": "#/definitions/StorageTaskAssignmentReport",
+          "description": "The storage task assignment report"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Represents the provisioning state of the storage task assignment.",
+          "enum": [
+            "ValidateSubscriptionQuotaBegin",
+            "ValidateSubscriptionQuotaEnd",
+            "Creating",
+            "Succeeded",
+            "Deleting",
+            "Canceled",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "runStatus": {
+          "$ref": "./common.json#/definitions/StorageTaskReportProperties",
+          "description": "Run status of storage task assignment"
+        }
+      },
+      "required": [
+        "taskId",
+        "enabled",
+        "description",
+        "executionContext",
+        "report"
+      ],
+      "description": "Properties of the storage task assignment."
+    },
+    "StorageTaskAssignmentExecutionContext": {
+      "type": "object",
+      "properties": {
+        "target": {
+          "$ref": "#/definitions/ExecutionTarget",
+          "description": "Execution target of the storage task assignment"
+        },
+        "trigger": {
+          "$ref": "#/definitions/ExecutionTrigger",
+          "description": "Execution trigger of the storage task assignment"
+        },
+        "constraints": {
+          "$ref": "#/definitions/ExecutionConstraints",
+          "description": "Execution constraints of the storage task assignment"
+        }
+      },
+      "required": [
+        "target",
+        "trigger"
+      ],
+      "description": "Execution context of the storage task assignment."
+    },
+    "ExecutionTarget": {
+      "type": "object",
+      "properties": {
+        "prefix": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of target prefixes"
```

Comment:
>    "description": "List of target prefixes" [](http://example.com/codeflow?start=6&length=44)

doc clarity: I can't yet figure out what we are talking about prefixes of? Storage accounts? Execution nodes?

Reply:
I assume these are the ones to 'include'?

Reply:
Tricia : "List of object prefixes that need to be included for task execution"

Reply:
Fixed